src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useContext, useState } from 'react';
+import { TodoItem } from '../TodoItem/TodoItem';
+import {
+  TodosContext,
+} from '../TodosContextProvider/TodosContextProvider';
+import { Filters } from '../Filters/Filters';
+import { FilterKey } from '../../types/FilterKey';
+import { Todo } from '../../types/Todo';
+
+function getFilteredTodos(key: FilterKey, todos: Todo[]) {
+  switch (key) {
+    case FilterKey.All:
+      return todos;
+    case FilterKey.Active:
+      return todos.filter(({ completed }) => !completed);
+    case FilterKey.Completed:
+      return todos.filter(({ completed }) => completed);
+    default:
+      return todos;
+  }
+}
+
+export const TodoList: React.FC = () => {
+  const [filterKey, setFilterKey] = useState<FilterKey>(FilterKey.All);
+  const { todos, setTodos } = useContext(TodosContext);
+
+  const areAllTodosCompleted = todos.every(({ completed }) => completed);
```

Comment:
I know that it is lexically correct to use "are", but it is customary to use "iz" in such cases`
```suggestion
  const isAllTodosCompleted = todos.every(({ completed }) => completed);
```